src/CMakeLists.txt

```diff
@@ -103,7 +103,7 @@ set_target_properties(glfw PROPERTIES
 target_compile_definitions(glfw PRIVATE _GLFW_USE_CONFIG_H)
 target_include_directories(glfw PUBLIC
                            "$<BUILD_INTERFACE:${GLFW_SOURCE_DIR}/include>"
-                           "$<INSTALL_INTERFACE:${CMAKE_INSTALL_FULL_INCLUDEDIR}>")
+                           "$<INSTALL_INTERFACE:${GLFW_INSTALL_INCLUDEDIR}>")
```

Comment:
Ok so this is the one divergence from the original install behavior.  I can revert this to full path if desired, it's easiest to explain via snippets of installed configuration files.

#### current `master` example

```cmake
set_target_properties(glfw PROPERTIES
  INTERFACE_INCLUDE_DIRECTORIES "/tmp/glfw_master/include"
  INTERFACE_LINK_LIBRARIES "/usr/lib64/librt.so;/usr/lib64/libm.so;dl;/usr/lib64/libX11.so;-lpthread"
)
```

#### this branch defaults

```cmake
set_target_properties(glfw PROPERTIES
  INTERFACE_INCLUDE_DIRECTORIES "${_IMPORT_PREFIX}/include"
  INTERFACE_LINK_LIBRARIES "/usr/lib64/librt.so;/usr/lib64/libm.so;dl;/usr/lib64/libX11.so;-lpthread"
)
```

So unless an absolute path is specified, this means that the `$<INSTALL_INTERFACE:...>` include being used here makes it so that the glfw installation can be relocated.  For reference, in the `glfw3Targets-{config}.cmake` file you will get a relocatable library location:

```cmake
set_target_properties(glfw PROPERTIES
  IMPORTED_LINK_INTERFACE_LANGUAGES_NOCONFIG "C"
  IMPORTED_LOCATION_NOCONFIG "${_IMPORT_PREFIX}/lib64/libglfw3.a"
  )
```

and in the event that somebody defines an absolute path with `-DGLFW_INSTALL_INCLUDEDIR`, there's nothing that can be done.